apps/lukso-manager/src/runner/config.go

```diff
@@ -0,0 +1,33 @@
+package runner
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/go-yaml/yaml"
+)
+
+type NetworkConfig struct {
+	GENESISTIME       int    `yaml:"GENESIS_TIME"`
```

Comment:
lets do `genesisTime` and consecutively with others

Reply:
```suggestion
	genesisTime       int    `yaml:"GENESIS_TIME"`
```

Are you sugessting this?